lib/actions/assign.js

```diff
@@ -20,6 +20,13 @@ const checkAssignee = async (context, issueNumber, assignee) => {
   return isValidAssignee(checkResult) ? assignee : null
 }
 
+const mapSpecialAssignee = async (payload, assignee) => {
```

Comment:
just a nitpick, does this function needs to be `async`?

Reply:
You are absolutely right this is not needed anymore.